cypress/e2e/certificate/residential_air_conditioner.cy.ts

```diff
@@ -0,0 +1,161 @@
+import {WorkBook} from "xlsx";
+
+import {DataExcel, SheetName} from "../../excel"
+import { 
+  FormSelector, 
+  ErrorSelector, 
+  ResultSelector, 
+  PostcodeState,
+  ErrorMessage
+} from "../../enum";
```

Comment:
in TS, dont do relative import, check out how do you do that using alias importing in the tsconfig.json